.github/workflows/release.yml

```diff
@@ -9,6 +9,11 @@ on:
 jobs:
   deploy:
     runs-on: ubuntu-latest
+    environment: release
```

Comment:
We are not using the environment. We can remove this IMO.

Reply:
@tetienne we need this actually for the connection to PyPi. They are tied to an environment.